### ISSUE TITLE
Assignment 24

### ISSUE THREAD

**Issue by twhitacre** [tiy-atl-js-sept-2015/Assignments#660]
@amitshah86 this is your issue for *Assignment 24*. Please comment below with the required deliverable(s) and close this issue when you are finished.

**Comment by amitshah86** [comment]
http://insidious-powder.surge.sh/#/

**Comment by amitshah86** [comment]
https://github.com/amitshah86/AngularMusic

**Comment by jisaacks** [comment]
Thanks Amit, the assignment was to have some sort of CRUD implementation (creating, editing, deleting) entries. I do not see that here?

**Comment by amitshah86** [comment]
Yes...I still have work left to do. I am going to work on all assignments
that need work this week and next week to get caught up. Thank You!

**Comment by jisaacks** [comment]
Sounds good! :+1:
